### ISSUE TITLE
Can't access preferences file

### ISSUE THREAD

**Issue by pheyvaer** [solid/node-solid-server#1366]
When I go to `/profile` on my POD (`https://ash.runi.ilabt.imec.be/profile/`) I get the following error in the console `index.js:1405 GET https://ash.runi.ilabt.imec.be/settings/prefs.ttl 403 (User Unauthorized)` together with the following warning `signin.js:325 You are not authorized to read your preferences file. This may be because you are using an untrusted web app.`. I manually added `https://ash.runi.ilabt.imec.be` to the trusted apps, but that doesn't make a difference.

My config.json is 
```
{
  "root": "/users/pheyvaer/ash/data",
  "port": "8443",
  "serverUri": "https://ash.runi.ilabt.imec.be",
  "webid": true,
  "mount": "/",
  "configPath": "./config",
  "configFile": "./config.json",
  "dbPath": "./.db",
  "sslKey": "ssl-key.pem",
  "sslCert": "ssl-cert.pem",
  "multiuser": false,
  "server": {
    "name": "ash.runi.ilabt.imec.be",
    "description": "",
    "logo": ""
  }
}
```
Solid version 5.2.2

**Comment by jaxoncreed** [comment]
What does your webid look like?

**Comment by pheyvaer** [comment]
It's `https://ash.runi.ilabt.imec.be/profile/card#me`

**Comment by megoth** [comment]
You're trying to access the resource from a web app running on https://ash.runi.ilabt.imec.be, right?

**Comment by pheyvaer** [comment]
Yes, I'm opening the url `https://ash.runi.ilabt.imec.be/profile/` in my browser.

**Comment by pheyvaer** [comment]
@RubenVerborgh

**Comment by jaxoncreed** [comment]
Do you have access to the `.acl` file for prefs.ttl?

**Comment by pheyvaer** [comment]
This is resolved together with #1351 so I think that the problem of #1351 was the cause of this issue too.
